app/api/migrations/migrations/41-change-system-translation-label/index.js

```diff
@@ -7,18 +7,8 @@ export default {
 
   async up(db) {
     process.stdout.write(`${this.name}...\r\n`);
-    const translations = await db
+    await db
       .collection('translations')
-      .find({})
-      .toArray();
-
-    translations.forEach(async trans => {
-      const context = trans.contexts.find(ctxt => ctxt.id === 'System');
-      if (context) {
-        context.label = 'User Interface';
-
-        await db.collection('translations').updateOne({ _id: trans._id }, { $set: { ...trans } });
-      }
-    });
+      .updateMany({ 'contexts.id': 'System' }, { $set: { 'contexts.$.label': 'User Interface' } });
```

Comment:
@grafitto this looks much better. still for the next time please make sure you test more "real" fixtures. the contexts on your fixtures only contain ONE context.

Reply:
Noted